tardis/montecarlo/tests/test_montecarlo_line_scatter.py

```diff
@@ -0,0 +1,20 @@
+import os
+from ctypes import CDLL, c_double, c_longdouble
```

Comment:
reiterated from @aoifeboyle we do not need a c_longdouble. As I mentioned, double is more than big enough to store our values.